packages/breadcrumb/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+
+import { separator } from "../style";
+export interface BreadcrumbProps {
```

Comment:
Not blocking, but we usually have an empty line between the imports and the component.